engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/TestBase.java

```diff
@@ -71,7 +71,7 @@ protected static EnsoContext leakContext(Context ctx) {
    */
   protected static Value executeInContext(Context ctx, Callable<Object> callable) {
     // Force initialization of the context
-    ctx.eval("enso", "42");
+    ctx.eval("enso", "value = 0");
```

Comment:
I'm curious - why is the code here changed?

Reply:
Because `42` is not a valid Enso code.

Reply:
It would even be better to replace this line with:
```java
ctx.initialize(LanguageInfo.ID);
```

Reply:
There were [failing tests](https://github.com/enso-org/enso/pull/9864#discussion_r1590766402).

Reply:
Could you please try to replace it with `ctx.initialize(LanguageInfo.ID);`?